shared/js/contacts/import/utilities/overlay.js

```diff
@@ -177,6 +236,8 @@ var utils = window.utils || {};
       overlay.classList.add('no-opacity');
       overlay.classList.add('hide');
       utils.overlay.isShown = false;
+      window.dispatchEvent(new CustomEvent('overlayhidden'));
+      deleteOverlay();
     });
   };
   utils.overlay.hide = function ov_hide() {
```

Comment:
No need to overload `window.utils`. I would use `Overlay` as something independent (`window.Overlay`). No need to add complicated namespaces.